src/components/charts/NetWorthHistogram.tsx

```diff
@@ -20,24 +20,41 @@ C.register(
   LineController,
 );
 
-export type NetWorthHistogramProps = {
+export type NetWorthHistogramProps =
```

Comment:
users can now pass either only assetsGuid or liabilitiesGuid or both at the same time.